### ISSUE TITLE
Feature Request: Maintain tiles manually adjusted positions when copying

### ISSUE THREAD

**Issue by Zefk** [christopho/solarus-quest-editor#176]
As you can see...the manual positions I entered for the tiles were not copied. That is why maintaining the tiles manually adjusted positions when copying would be a good feature.
![maintain position when copying](https://cloud.githubusercontent.com/assets/19560764/15698441/81435a1c-2779-11e6-9897-a42fb0e8f814.png)

**Comment by christopho** [comment]
Did you manually enter coordinates that are not multiples of 8?

**Comment by Zefk** [comment]
That is correct. That is why I opened this issue #180. It could fix the problem. It takes a long time to make them all perfectly 8x8 compatible.

**Comment by alexander-b** [comment]
I'm not convinced this is "fixable" since Solarus needs these things to be 8x8. Perhaps we should instead work on the UX of the quest editor, by making sure the program makes the user aware of these issues. Right now it will apparently just discards user input without any explanation. That's really bad.

**Comment by Zefk** [comment]
Yes, but why not make it more than just 8x8? (I requested this, but I do not think it was understood...) I have already reformatted for my [project](https://github.com/Zefk/Fairyolica-World) for Solarus to 8x8 and fixed this, but I still think it would be a good idea.
